src/controllers/usersController.js

```diff
@@ -333,3 +333,16 @@ export const deleteNotification = async (req, res) => {
     return errorRes(res, 500, res.__('Error while deleting  notification'));
   }
 };
+
+
+
+
+
+
+
+
+  
+
+
+
+
```

Comment:
remove these empty lines. let's have only empty line. you should follow eslint rules